target-platform/config/kura.target-platform.build.properties

```diff
@@ -26,6 +26,8 @@ com.h2database.h2.version=1.4.192
 slf4j.api.version=1.7.25
 jcl.over.slf4j.version=1.7.25
 log4j.version=2.8.2
+log4j.log4j.version=1.2.17
```

Comment:
To add those dependencies we need CQs. I've not checked but I believe we don't have them for those bundles.
Is there a specific reason to have them?

Reply:
When i import kura source into eclipse. Some Testing projects have compile errors in eclipse due to lack this bundles. And i can't compile in eclipse. Only maven.